packages/app/src/components/SearchBar/index.js

```diff
@@ -16,16 +16,19 @@ const SearchBar = () => {
 
 
   function findSearchResults(searchTerm) {
-    const filteredNoms = NominationsData.filter((nomination) => {
-      return [
-        formatSearch(nomination.providerName),
-        formatSearch(nomination.patientName),
-        formatSearch(nomination.nominationName),
-        formatSearch(nomination.representativeName),
-      ].some((nom) => nom.includes(searchTerm));
-    });
-    setSearchResultData(filteredNoms);
-
+    let filteredNoms = []
+    console.log('NominationsData ', NominationsData)
+    if(NominationsData) {
+      filteredNoms = NominationsData.filter((nomination) => {
+        return [
+          formatSearch(nomination.providerName),
+          formatSearch(nomination.patientName),
+          formatSearch(nomination.nominationName),
+          formatSearch(nomination.representativeName),
+        ].some((nom) => nom.includes(searchTerm));
+      });
+      setSearchResultData(filteredNoms);
+    } else
```

Comment:
I would recommend having another curly brace block after the `else` statement and ended after the ternary just for consistency with the `if` block above.

like 
```javascript 
} else {
  filteredNoms.length < 1
      ? setShowErrorMessage(true)
      : setShowErrorMessage(false);
}